packages/web/src/app/doc-pages/about/tutorial/react-tutorial/react-tutorial.component.html

```diff
@@ -0,0 +1,95 @@
+<!--BASIC TUTORIAL-->
+<h3 class="e-title-small e-mt-40 e-mb-0">Set up the basics</h3>
+<ol class="e-list e-list--numbers e-mt-40">
+  <li>
+    <div class="e-title-xsmall">Install CSS library</div>
+    <p class="e-mt-8">
+      Install the
+      <a [routerLink]="['/components/css-library']" class="e-link e-link--inline">CSS library</a> to get the
+      basic components and styling.
+    </p>
+    <p class="e-my-8 e-text-md">
+      <strong
+        >The packages acts like any other external dependency you install and update in your project.
+      </strong>
+    </p>
+    <app-code-viewer [tabs]="['Bash']" [fileName]="'Terminal'" [activeCode]="codeStep1" hideTabs />
+  </li>
+  <li class="e-mt-24">
+    <div class="e-title-xsmall">Add imports</div>
+    <p class="e-my-8">Import the stylesheet and script.</p>
+    <app-code-viewer [tabs]="['Typescript']" [fileName]="'main.tsx'" [activeCode]="codeStep2" hideTabs />
+  </li>
+  <li class="e-mt-24">
+    <div class="e-title-xsmall">Start using classes</div>
+    <p class="e-my-8">
+      Choose between multiple
+      <a [routerLink]="['/tools/utility-classes']" class="e-link e-link--inline">brand utilities</a> and
+      <a [routerLink]="['/components']" class="e-link e-link--inline">components</a>. The components have
+      multiple variants and options to fit different needs.
+    </p>
+    <app-code-viewer [tabs]="['HTML']" [fileName]="'App.tsx'" [activeCode]="codeStep3" hideTabs />
+  </li>
+  <li class="e-mt-24">
+    <div class="e-title-xsmall">Add CSS-reset<span class="e-tag e-ml-8">Optional</span></div>
+    <p class="e-mt-8">
+      Import the Elvis CSS reset if you wish to apply a generic CSS reset to your application.
+    </p>
+    <p class="e-my-8 e-text-md">
+      <strong
+        >Be aware that you may break styles in your app if you import this in an existing application, since
+        you may rely on browser defaults without knowing it. This is why this import is separate from the
+        required Elvis import.</strong
+      >
+    </p>
+    <app-code-viewer [tabs]="['Typescript']" [fileName]="'main.ts'" [activeCode]="codeStep4" hideTabs />
+  </li>
+</ol>
+
+<!--ADVANCED TUTORIAL-->
+<h3 class="e-title-small e-mt-64">Add an advanced component</h3>
+<ol class="e-list e-list--numbers e-mt-40">
+  <li>
+    <div class="e-title-xsmall">Install component</div>
+    <p class="e-mt-8">
+      Install the
+      <a [routerLink]="['/components/datepicker']" class="e-link e-link--inline">datepicker component.</a>
+    </p>
+    <p class="e-my-8 e-text-md">
+      <strong
+        >The packages acts like any other external dependency you install and update in your project.
```

Comment:
```suggestion
        >The packages act like any other external dependency you install and update in your project.
```

Reply:
Maybe we can create a file containing string that appear in multiple tutorials, so that we only have to write it once 🤔